app/models/validators/utaac_decision_validator.rb

```diff
@@ -12,6 +13,6 @@ class UtaacDecisionValidator < SimpleDelegator
   validates :tribunal_decision_category, presence: true
   validates :tribunal_decision_decision_date, presence: true, date: true
   validates :tribunal_decision_judges, presence: true
-  validates :tribunal_decision_sub_category, presence: true
+  validates :tribunal_decision_sub_category, presence: true, tribunal_decision_sub_category: true
```

Comment:
This line looks a bit weird to me. Could the `tribunal_decision_sub_category: true` bit maybe explain _what_ is being validated here? Presumably that the subcategory matches the parent category?